test/parse_stream.jl

```diff
@@ -151,9 +149,80 @@ end
     @test last_child_position(st, position(st)) == ParseStreamPosition(7, 2)
 end
 
+# This is coppied from InlineStrings.jl instead of depending
+# on InlineStrings for compat with Julia 1.2 and earlier:
```

Comment:
Surely there must be some smaller way to define a string that doesn't have a `pointer` on it? Something like https://github.com/JuliaLang/julia/blob/a993cd8f81a6bc02a88ee5cf036f4e29c36d5580/stdlib/Test/src/Test.jl#L2200-L2210?